@stellar/typescript-wallet-sdk/src/walletSdk/Types/auth.ts

```diff
@@ -91,3 +94,17 @@ export type SignChallengeTxnResponse = {
   transaction: XdrEncodedTransaction;
   networkPassphrase: NetworkPassphrase;
 };
+
+export type AuthHeaderClaims = {
+  account: string;
+  home_domain: string;
+  webAuthEndpoint: string;
```

Comment:
should this be `web_auth_endpoint` instead?

Reply:
yes, good 👁️  sir
https://github.com/stellar/typescript-wallet-sdk/pull/124/commits/84307862127c69aa96429093fb95b5a69fb910a3